be/src/vec/data_types/serde/data_type_bitmap_serde.cpp

```diff
@@ -52,17 +53,17 @@ Status DataTypeBitMapSerDe::deserialize_one_cell_from_json(IColumn& column, Slic
     return Status::OK();
 }
 
-Status DataTypeBitMapSerDe::write_column_to_pb(const IColumn& column, PValues& result, int start,
-                                               int end) const {
+Status DataTypeBitMapSerDe::write_column_to_pb(const IColumn& column, PValues& result,
+                                               int64_t start, int64_t end) const {
     auto ptype = result.mutable_type();
     ptype->set_id(PGenericType::BITMAP);
     auto& data_column = assert_cast<const ColumnBitmap&>(column);
-    int row_count = end - start;
+    auto row_count = cast_set<int>(end - start);
```

Comment:
why here need cast ?  use int64_t?

Reply:
result.mutable_bytes_value()->Reserve(row_count);
`Reserve` need int argument, 
and the function is call as block batch, so int of row_count is ok.